hack/e2e/run-e2e.sh

```diff
@@ -119,7 +119,7 @@ mkdir -p "${ROOT_DIR}/tests/e2e/out"
 # would create CPUs-1 nodes and saturate the testing server
 RC_GINKGO2=0
 LABEL_FILTERS="!(upgrade)"
-if [ ! -z "${FEATURE_TYPE+x}" ]; then
+if [ -n "${FEATURE_TYPE+x}" ]; then
```

Comment:
Could you please use


```suggestion
if [ "${FEATURE_TYPE:-}" ]; then
```